cfssl serve with multiroot not working

i am running the latest (tag, 5 days old) docker version from https://hub.docker.com/r/cfssl/cfssl/

one instance as multiroot - that works fine
and one with cfssl serve, which fails immediately with the following error

```
panic: interface conversion: http.Handler is *api.HTTPHandler, not api.HTTPHandler

goroutine 1 [running]:
panic(0xa96240, 0xc8204e5b40)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/cloudflare/cfssl/cli/serve.glob.func2(0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/cloudflare/cfssl/cli/serve/serve.go:150 +0x158
github.com/cloudflare/cfssl/cli/serve.registerHandlers()
	/go/src/github.com/cloudflare/cfssl/cli/serve/serve.go:250 +0x1da
github.com/cloudflare/cfssl/cli/serve.serverMain(0xc82000a2b0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7ffee0ddadb7, ...)
	/go/src/github.com/cloudflare/cfssl/cli/serve/serve.go:297 +0x635
github.com/cloudflare/cfssl/cli.Start(0xc8200e8480, 0x0, 0x0)
	/go/src/github.com/cloudflare/cfssl/cli/cli.go:142 +0xb84
main.main()
	/go/src/github.com/cloudflare/cfssl/cmd/cfssl/cfssl.go:80 +0x5ee
```

My experiment with the latest image fails with such error when I enable authsign endpoint by adding "auth_key" to config file of cfssl serve:

```
2017/07/28 10:25:32 [INFO] endpoint '/api/v1/cfssl/scan' is enabled
2017/07/28 10:25:32 [DEBUG] getHandler for authsign
panic: interface conversion: http.Handler is *api.HTTPHandler, not api.HTTPHandler

goroutine 1 [running]:
github.com/cloudflare/cfssl/cli/serve.glob..func2(0xa19f2b, 0x11, 0xc4208250f0, 0x1)
	/go/src/github.com/cloudflare/cfssl/cli/serve/serve.go:150 +0x20a
github.com/cloudflare/cfssl/cli/serve.registerHandlers()
	/go/src/github.com/cloudflare/cfssl/cli/serve/serve.go:250 +0x156
github.com/cloudflare/cfssl/cli/serve.serverMain(0xc420010130, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7ffc13ba7ec2, ...)
	/go/src/github.com/cloudflare/cfssl/cli/serve/serve.go:297 +0x3a7
github.com/cloudflare/cfssl/cli.Start(0xc4200f62a0, 0xc4200f62a0, 0xc420045f68)
	/go/src/github.com/cloudflare/cfssl/cli/cli.go:143 +0x335
main.main()
	/go/src/github.com/cloudflare/cfssl/cmd/cfssl/cfssl.go:83 +0x8a9
```

--- a/cli/serve/serve.go
+++ b/cli/serve/serve.go
@@ -147,7 +147,7 @@ var endpoints = map[string]func() (http.Handler, error){
 		}
 
 		if conf.CABundleFile != "" && conf.IntBundleFile != "" {
-			sh := h.(api.HTTPHandler).Handler.(*signhandler.AuthHandler)
+			sh := h.(*api.HTTPHandler).Handler.(*signhandler.AuthHandler)
 			if err := sh.SetBundler(conf.CABundleFile, conf.IntBundleFile); err != nil {
 				return nil, err
 			}

